index.d.ts

```diff
@@ -17,33 +17,61 @@ export interface SnapshotOptions {
 	id?: string;
 }
 
+// When an assert fail, the message string will be shown in the error
```

Comment:
Would this ever be shown to the user? I agree that we shouldn't document it for each assertion though.